database/controllers/index.js

```diff
@@ -4,36 +4,53 @@ module.exports = {
     dbm,
 };
 
-
 /*
 
 //Mock JSON object with User + Registration info.
 //To test that proper creation works.
 var tempD = {
-    "badge": '1010101010',
-    "firstName": "We are Real",
-    "lastName": "But We are Also fake",
-    "status": "User",
-    "email": "eeeetssssss@know.com",
-    "signature": "this is a signature",
-    "date": "01-04-2000",
+    "badge": '9999999',
+    "firstName": "Harry",
+    "lastName": "Someone",
+    "email": "lalala@lololo.com",
+    "signature": "Harry Someone",
     "phone": "(555)1234-5555",
-    "ecName": "dude",
-    "ecRelation": "Yo",
+    "ecName": "Some Guy",
+    "ecRelation": "Someone",
     "ecPhone": "(313)123-4220"
 }
 
+
 //This is just a simple test to create a user.
 //The .done() call is used to wait for the DB interaction to finish before
 // executing the if statement.
 dbm.createUser(tempD).done(function(test){
    if(test.result) {
-       //dbm.deleteUser(tempD.badge).done();  --Uncomment if you want to delete user after creation
+       //dbm.deleteUser(tempD.badge).done();
        console.log(test.detail);
    }
    else
        console.log(test.detail);
    process.exit();
 });
 
+
 */
+
+//For testing get user function
+/*var one = '2000-01-01';
+var two = '2018-02-04';
+
+dbm.getUser(one, two).done(function(test){
+    if(test != 0) {
+        console.log(test.length);
+        for(var i = 0; i < test.length; ++i){
+                console.log(test[i]);
+        }
+    }
+    else
+        console.log(test);
+    process.exit();
+});
+
```

Comment:
Could we remove this commented out testing code? Perhaps I have a better idea, what if we create a folder called `db_tests` in the root dir and put these tests as mocha tests. This is fine because they just wont run on travis but we will still have them and it will probably make our lives easier when we would want to run quick mocha tests manually to check the DB is all good.

Reply:
Sounds Good.